tests/e2e/workloads/test_data_consistency.py

```diff
@@ -50,6 +51,10 @@ def test_log_reader_writer_parallel(project, tmp_path):
     # get deployment dict for the reproducer logwriter workload
     with open(constants.LOGWRITER_CEPHFS_REPRODUCER, "r") as deployment_file:
         deploy_dict = yaml.safe_load(deployment_file.read())
+    # if we are running in disconnected environment, we need to mirror the
+    # container image first, and then use the mirror instead of the original
+    if config.DEPLOYMENT.get("disconnected"):
```

Comment:
You can even omit this condition here as it is also part of the [`update_container_with_mirrored_image`](https://github.com/red-hat-storage/ocs-ci/blob/bc284be38fb1dc725333253f0d481ac13d596a62/ocs_ci/utility/utils.py#L2987) function. But if you want to leave it here for better readability, I'm ok with that.

Reply:
Yes, I wanted to be more explicit.